Protacon.RxMq.AzureServiceBusLegacy/Topic/AzureBusTopicPublisher.cs

```diff
@@ -75,6 +76,11 @@ public Task SendAsync(T message, string topicName)
                     ContentType = "application/json"
                 };
 
+                if (message is IHasCorrelationId correlationMessage)
+                {
+                    brokeredMessage.CorrelationId = correlationMessage.CorrelationId ?? brokeredMessage.CorrelationId;
```

Comment:
Why are we assigning correlationId from brokeredMessage to brokeredMessage? Shouldn't line 79 have null check or should ?? brokeredMessage.CorrelationId be something else?

Reply:
`brokeredMessage.CorrelationId = correlationMessage.CorrelationId ?? brokeredMessage.CorrelationId;` is just a quick way to set correlation ID from `message` or leave it as-is. This could also be `brokeredMessage.CorrelationId = correlationMessage.CorrelationId` but in that case the netcore-side should also be changed. I'm not sure if CorrelationId is populated by default so I'm not sure if there are side effects if this would be set to null.

Null check is not needed here. correlation message is already checked